### PR TITLE
Fix up fresh machine script

### PR DESCRIPTION
**Pull request by C-alcium**
I setup a new mac recently, I had to setup a new mac and ran into some issues. Particularly with Pyenv + yarn being installed, for whatever reason that messed up the package.json file of the project I installed in the dir of.

### CODE REVIEW

File: scripts/mac_intel_install.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/zsh
+;#!/bin/zsh
```
Comment (gone) [inline scripts/mac_intel_install.sh:1]: I think you need this - potentially it should be bash if you're worried about people not having zsh?

File: scripts/mac_intel_install.sh [hunk 2]
```diff
@@ -56,25 +52,21 @@ echo "eval \"source \$(brew --prefix nvm)/nvm.sh\"" >> ~/.zshrc
...
-pyenv global 3.10.2
 
 echo "${prefix}Attempting to change nvm version to v16.14.0(default)"
 nvm install v16.14.0
 nvm alias default v16.14.0
 nvm use default
 
 echo "${prefix}Attempting to install poetry"
-curl -sSL https://install.python-poetry.org | python3 -
+curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py | python - --preview
```
Comment (gone) [inline scripts/mac_intel_install.sh:69]: I think this needs to get updated now that poetry 1.2 is offical?

File: scripts/mac_intel_install.sh (hunk 1 above)
Comment (C-alcium) [reply scripts/mac_intel_install.sh:1]: Ah you're right

File: scripts/mac_intel_install.sh (hunk 2 above)
Comment (C-alcium) [reply scripts/mac_intel_install.sh]: Reverted now

File: scripts/mac_intel_install.sh (hunk 1 above)
Comment (C-alcium) [reply scripts/mac_intel_install.sh]: Removed

**Pull request merged**